src/symbol.py

```diff
@@ -0,0 +1,15 @@
+class SymbolOptions:
+    def get_symbols(self, user_input):
```

Comment:
You could treat this more like a config file, and allow messages to also depend on it, that way you do not end up having to maintain symbol options in 2 places.